### ISSUE TITLE
Ios 14.7.1 not working

### ISSUE THREAD

**Issue by Unsuspect** [donato-fiore/GameSeagull#59]
I jailbroke with Taurine use Sileo and zebra managers (iphone 12). 

The problem is I have the tweak installed just doesn't do anything I had to also downgrade game pigeon to a 2.2.2 for it to work on jailbreak and tweaks still won't work. None of tweak work for me sadly. Thanks for reading...

**Comment by pandadevv5495** [comment]
just checking, did you enable the cheats in settings..

**Comment by Unsuspect** [comment]
Yes I even used choicy to disabled and re-enable, downloaded it as a deb and ipa tweaks didn't work for me.
